pepys_import/file/file_processor.py

```diff
@@ -1,18 +1,36 @@
+import inspect
+import importlib.util
 import json
 import os
 import shutil
+import sys
 
 from datetime import datetime
 from stat import S_IREAD
+from pathlib import Path
 
 from pepys_import.core.store.data_store import DataStore
 from pepys_import.core.store.table_summary import TableSummary, TableSummarySet
 from pepys_import.file.highlighter.highlighter import HighlightedFile
+from pepys_import.file.importer import Importer
+
+ROOT_PATH = Path(__file__).parent.parent.parent  # Main directory
```

Comment:
ooh, this seems fragile.  Is there a way of doing it that doesn't rely on us being 3 folders deep?

Reply:
I created `path.py` at the root of the repository. So, we don't need to use Path() library to go to the file's parents 3 times:

https://github.com/debrief/pepys-import/pull/172/commits/5d0b3e88ac56a5691977ad2a4a3b99ae9de41c23